pandas/core/arrays/string_.py

```diff
@@ -50,16 +47,8 @@ class StringDtype(ExtensionDtype):
     StringDtype
     """
 
-    @property
-    def na_value(self) -> "Scalar":
-        """
-        StringDtype uses :attr:`numpy.nan` as the missing NA value.
-
-        .. warning::
-
-           `na_value` may change in a future release.
-        """
-        return np.nan
+    #: StringDtype.na_value uses pandas.NA
```

Comment:
is the colon here meaningful or a typo?

Reply:
That's how sphinx extracts docs for class attributes: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute